### ISSUE TITLE
Run command only after init

### ISSUE THREAD

**Issue by mittalyashu** [gitpod-io/gitpod#2015]
As stated in this [article](https://www.gitpod.io/blog/gitpodify/):

```yaml
tasks:
  - init: |
      touch /tmp/.npm-lock
      npm install
      rm /tmp/.npm-lock
    command: npm run server
  - init: sleep 1 && while [ -f /tmp/.npm-lock ]; do sleep 1; done
    command: npm run client
```

We need to this little hack to run a client process after installing the packages.

It would be nice to have a proper support like

```yaml
tasks:
  - init: npm install
    command: npm run server
  - command: npm run client
    after: init # or something like that
```

**Comment by svenefftinge** [comment]
We have `gp sync-done <name>`  resp. `gp sync-await <name>` for that, which allows for even more fine grained management of concurrent processes.
E.g.:
```
tasks:
  - init: |
      npm install
      gp sync-done npm-install
    command: npm run server
  - command: |
      gp sync-await npm-install
      npm run client
```

**Comment by mittalyashu** [comment]
@svenefftinge I tried to find more about `gp sync` in the documentation, didn't find anything related to it.
